code/modules/clothing/head/misc.dm

```diff
@@ -260,3 +260,28 @@
 		rad = 0
 	)
 
+/obj/item/clothing/head/metaljacket
+	name = "sergeant hat"
+	desc = "A rather generic sergeant-like hat."
+	icon_state = "metaljacket"
+	item_state = "metaljacket"
+	price_tag = 400
+
+/obj/item/clothing/head/inhailer
+	name = "odd looking helmet"
+	desc = "A confusingly complex helmet. It is capable of protecting you so it's more useful than being a simple decoration."
+	icon_state = "inhailer"
```

Comment:
fix these too

Reply:
done, also changed the sarge hat into a ranger hat